src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -89,6 +89,12 @@ public class Win32PlatformOptions
         /// This is recommended if you need to use AcrylicBlur or acrylic in your applications.
         /// </remarks>
         public bool UseWindowsUIComposition { get; set; } = true;
+
+        /// <summary>
+        /// When <see cref="UseWindowsUIComposition"/> enabled, create rounded corner blur brushes
+        /// If set to zero the brushes will be created using default settings (sharp corners)
+        /// </summary>
+        public float CompositionBackdropCornerRadius { get; set; }
```

Comment:
Does it mean that by default avalonia apps won't have rounded corners?

Reply:
Yes, sharp corners just as it's now. 
I am still thinking whether to add this as a window property but then that might be only possible using something similar to what @kekekeks suggested above, do you think it's worth implementing as part of this PR?

Reply:
I am sorry for missing this question. 
I don't like that this property is missleading. It says "If set to zero the brushes will be created using default settings (sharp corners)", but in reality default setting now depends on windows OS - it's sharp on 10 and round on 11. Is there any way to get its default value from the API?
We can keep this property nullable and apply value only when it's not null.